src/integrity-test/validate-package-versions-match.js

```diff
@@ -0,0 +1,173 @@
+const fs = require('fs');
+const chalk = require('chalk');
+const path = require('path');
+
+/**
+ * @param {string} vendorDir
+ * @returns {object}
+ */
+function getComposerPackagesConfig(vendorDir) {
+  let packageInfo = {};
+
+  if (!fs.existsSync(vendorDir)) {
+    throw new Error('Vendor directory not found');
+  }
+
+  const composerLockPath = path.join(vendorDir + '/..', 'composer.lock');
+
+  let composerLockData;
+  try {
+    composerLockData = JSON.parse(fs.readFileSync(composerLockPath));
+  } catch (e) {
+    throw new Error(`Failed to read or parse composer.lock file: ${e.message}`);
```

Comment:
Does `e.message` include the path to the composer.lock file? If not, it would be helpful to add it.